### DISCUSSION TITLE
AWS Distributed Installation

### DISCUSSION THREAD

**Discussion by mtoback** [Security-Onion-Solutions/securityonion#7823] [Unsupported Versions] [answered]
Are there any documents, videos, blogs, etc that people know of that give a detailed description of setting up security onion on AWS in distributed mode? Everything I've seen either is standalone or is a very high level description. For instance, I happened to stumble on a post that says you need to enable password login on the manager so that soremote works, and right now I am stuck on docker working and can't figure out why it is failing: 

docker.service - Docker Application Container Engine
     Loaded: loaded (/lib/systemd/system/docker.service; enabled; vendor preset: enabled)
     Active: active (running) since Tue 2022-04-19 14:33:48 UTC; 4h 44min ago
TriggeredBy: ● docker.socket
       Docs: https://docs.docker.com
   Main PID: 15112 (dockerd)
      Tasks: 10
     Memory: 27.3M
     CGroup: /system.slice/docker.service
             └─15112 /usr/bin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock

Apr 19 18:46:05 ec2-34-211-46-212 dockerd[15112]: time="2022-04-19T18:46:05.275087656Z" level=error msg="Handler for POST /v1.41/images/create r>
Apr 19 18:46:21 ec2-34-211-46-212 dockerd[15112]: time="2022-04-19T18:46:21.193413471Z" level=warning msg="Error getting v2 registry: Get \"http>
Apr 19 18:46:21 ec2-34-211-46-212 dockerd[15112]: time="2022-04-19T18:46:21.193480062Z" level=info msg="Attempting next endpoint for pull after >
Apr 19 18:46:21 ec2-34-211-46-212 dockerd[15112]: time="2022-04-19T18:46:21.195800408Z" level=error msg="Handler for POST /v1.41/images/create r>

**Comment by dougburks** [comment, marked as answer]
Have you checked our AWS documentation?
https://docs.securityonion.net/en/2.3/cloud-ami.html

**Reply by mtoback** [reply]
Thank you dougburks, I will try this.
